### ISSUE TITLE
Screen readers do not announce descriptive information for the 'Change', 'Repair' and 'Remove' button in windows.

### ISSUE THREAD

**Issue by msftedad** [wixtoolset/issues#9324]
### WiX Version

7.0.0

### .NET or MSBuild or Visual Studio Version

.NET SDK (10.0.300), MS Build (18.6.11), visual Studio (2022 v17.14.37502.11)

### Repro Steps

**Pre-Requisite:** 
1. Turn on Screen Readers:
NVDA: Ctrl + Alt + N 
JAWS: Ctrl + Alt + J 
Narrator: Ctrl + win + Enter 
2. Verbosity:
NVDA: Default 
JAWS: Beginner, Highest 
Narrator: Default
3. Download the msi file '[DacFramework.msi installer for Windows](https://aka.ms/dacfx-msi)' and install it.

**Repro Steps:**

1. Open the installed 'Microsoft SQL Server Data-Tier Application Framework Setup' application in windows.
2. Press tab key to reach to the 'Next' button and activate it present on the dialog.
3. 'Change', 'Repair' and 'Remove' buttons will appear.
4. Verify whether screen readers do not announce descriptive information for the 'Change', 'Repair' and 'Remove' button in windows or not.

### Actual Result

Screen readers do not announce descriptive information for the 'Change', 'Repair' and 'Remove' button in windows.
Screen Reader announces as: Change button, Repair Button, Remove Button

**Refer Attachment:**
1.Screen readers do not announce descriptive information for the 'Change', 'Repair' and 'Remove' button in windows.png
2. Screen readers do not announce descriptive information for the 'Change', 'Repair' and 'Remove' button in windows.mp4

https://github.com/user-attachments/assets/cc593979-6ca6-4903-997c-d3b4414856da
<img width="1920" height="1228" alt="Image" src="https://github.com/user-attachments/assets/a6331f3f-f0c4-4d9d-8e75-fc933adf59e8" />

### Expected Result

Screen readers should announce descriptive information for the 'Change', 'Repair' and 'Remove' button in windows.
Screen Reader should announce: 
Change button
Microsoft SQL Server Data-Tier Application Framework has no independently selectable features.
Repair Button
Repairs errors in the most recent installation by fixing missing and corrupt files, shortcuts, and registry entries.
Remove button
Removes Microsoft SQL Server Data-Tier Application Framework from your computer.

### Acknowledgements

- [x] I acknowledge that this is a fully completed bug report. It is not a question or attempt to get help debugging my issue (because those should be [sent to Discussions](https://github.com/orgs/wixtoolset/discussions/new?category=questions)).
- [x] I am paying the [Open Source Maintenance Fee](https://opensourcemaintenancefee.org/) by [sponsoring the `wixtoolset` project](https://github.com/sponsors/wixtoolset) because I support the maintainers.

**Comment by barnson** [comment]
[MSI doesn't offer additional context for buttons other than bitmap buttons.](https://learn.microsoft.com/en-us/windows/win32/msi/accessibility) If you need more control, consider a bootstrapper application or embedded UI.
